src/main/webapp/app/core/sentry/sentry.error-handler.ts

```diff
@@ -1,5 +1,5 @@
 import { ErrorHandler, Injectable } from '@angular/core';
-import { captureException, dedupeIntegration, init } from '@sentry/angular';
+import { captureException, dedupeIntegration, init, browserTracingIntegration } from '@sentry/angular';
```

Comment:
Client style check is failing.

```suggestion
import { browserTracingIntegration, captureException, dedupeIntegration, init } from '@sentry/angular';
```